Windows release broken since 0.10.2 release

### What happened?

After installing the 0.11.1 release and trying to execute it we get an error related to a missing dependency.
```console
gefyra.exe
[ERROR] Could not create a valid configuration: No module named 'packaging'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "gefyra.__main__", line 5, in <module>
  File "gefyra.api", line 1, in <module>
  File "gefyra.api.bridge", line 8, in <module>
  File "gefyra.local.bridge", line 4, in <module>
  File "docker", line 2, in <module>
  File "docker.api", line 2, in <module>
  File "docker.api.client", line 10, in <module>
  File "docker.auth", line 7, in <module>
  File "docker.utils", line 3, in <module>
  File "docker.utils.decorators", line 4, in <module>
  File "docker.utils.utils", line 9, in <module>
ModuleNotFoundError: No module named 'packaging'
```

### What did you expect to happen?

We expect the execution  to show the help message, as happens if we try the gefyra-0.10.1-windows-x86_64 release
```console
 gefyra.exe
[INFO] There was no --endpoint argument provided. Connecting to a local Kubernetes node.
usage: gefyra [-h] [-d] [--kubeconfig KUBECONFIG] [--context CONTEXT] {up,run,bridge,unbridge,list,down,check,version,telemetry} ...

The Gefyra client. For more help please visit: https://gefyra.dev/reference/cli/

positional arguments:
  {up,run,bridge,unbridge,list,down,check,version,telemetry}
                        the action to be performed

optional arguments:
  -h, --help            show this help message and exit
  -d, --debug           add debug output
  --kubeconfig KUBECONFIG
                        path to kubeconfig file
  --context CONTEXT     context name from kubeconfig
```

### How can we reproduce it (as minimally and precisely as possible)?

Download the windows release 0.10.2 or  0.11.1
Unzip
Try to execute gefyra.exe
The execution ends with an error
````console
[ERROR] Could not create a valid configuration: No module named 'packaging'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "gefyra.__main__", line 8, in <module>
  File "gefyra.local.utils", line 6, in <module>
  File "docker", line 2, in <module>
  File "docker.api", line 2, in <module>
  File "docker.api.client", line 10, in <module>
  File "docker.auth", line 7, in <module>
  File "docker.utils", line 3, in <module>
  File "docker.utils.decorators", line 4, in <module>
  File "docker.utils.utils", line 9, in <module>
ModuleNotFoundError: No module named 'packaging'
```


### What Kubernetes setup are you working with?

<details>

```console
$ kubectl version
# paste output here
```

</details>


### OS version

<details>

```console
# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
BuildNumber  Caption                   OSArchitecture  Version
19044        Microsoft Windows 10 Pro  64-bit          10.0.19044

```

</details>


### Anything else we need to know?

Although the app looks a python app self-contained and shouldn't be affected, we have tried to test uninstalling a local installation of python with scoop, without luck.

--- a/client/pyoxidizer.bzl
+++ b/client/pyoxidizer.bzl
@@ -72,7 +72,7 @@ def make_win_exe():
 
     # windows
     exe.add_python_resources(exe.read_package_root(CWD, ["gefyra"]))
-    exe.add_python_resources(exe.pip_install(["--no-deps", "docker==6.0.0"]))
+    exe.add_python_resources(exe.pip_install(["docker==6.0.0"]))
     # certifi from version 2022.06.15.1 does not work
     exe.add_python_resources(exe.pip_install(["certifi==2022.06.15", "pywin32", "kubernetes", "tabulate", "cli-tracker"]))
     exe.windows_runtime_dlls_mode = "always"

